libs/@blockprotocol/graph/src/graph-module.json

```diff
@@ -163,14 +158,8 @@
       "respondedToBy": "aggregateEntitiesResponse",
       "sentOnInitialization": false,
       "data": {
-        "type": "object",
-        "properties": {
-          "operation": {
-            "description": "The aggregation operation to apply",
-            "$ref": "https://blockprotocol.org/types/modules/graph/aggregation-operation"
-          }
-        },
-        "required": ["operation"]
+        "description": "The aggregation operation to apply",
+        "$ref": "https://blockprotocol.org/types/modules/graph/0.3/schema/aggregation-operation"
```

Comment:
Shouldn't this be something like this?

```ts
"data": {
  "description": "The aggregation operation to apply",
  "type": "object",
  "$ref": "https://blockprotocol.org/types/modules/graph/0.3/schema/aggregation-operation",
  "properties": {
    "graphResolveDepths":  {
       "$ref": ...
    }
  }
}
```

(I think it would be clearer if `aggregation-operation` didn't include the `"operation"` key in it)

Reply:
Should be fixed in [Fix aggregation operation](https://github.com/blockprotocol/blockprotocol/pull/1034/commits/61e6b7a5d4ea03cba90759ae2f0128d9598e4396)

Reply:
Oh I see you had it in the actual aggregation-operation schema